src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -33,6 +33,30 @@ using namespace mlpack::ann;
 
 BOOST_AUTO_TEST_SUITE(LossFunctionsTest);
 
+/**
+ * Binary Cross Entropy with Logits Loss function test.
+ */
+BOOST_AUTO_TEST_CASE(BCEWithLogitsLossTest)
+{
+  arma::mat input, output, target;
+  BCEWithLogitsLoss<> module;
```

Comment:
Hey @mrityunjay-tripathi , You should `#include` your loss function because you compiler is not able to identify `BCEWitLogitsLoss` .

Reply:
ohh XD